jumpscale/packages/marketplace/frontend/data.js

```diff
@@ -97,6 +97,14 @@ const SECTIONS = {
         helpLink: "https://now10.threefold.io/docs",
         description: "Discourse is an open source Internet forum and mailing list management software application built to educate members about civil community engagement."
       },
+      meetings: {
+        name: "Meetings",
+        type: "meetings",
+        image: "./assets/meetings.png",
+        disable: false,
+        helpLink: "https://now10.threefold.io/docs",
```

Comment:
shouldn't it be https://now10.threefold.io/docs/video-conf/ ?

Reply:
Doesn't matter for me, you can change it to whatever suits best for you

Reply:
now10 is the commercial content, all of the apps listed in the marketplace should link to the commercial content, please update it to the meeting entry in now10